scripts/version-utils.js

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env node
+
+const { logger, spawnOrFail, shouldContinuePrompt, process } = require('./utilities.js');
+const prompt = require('prompt-sync')();
+const path = require('path');
+const fs = require('fs');
+
+const currentVersion = require('../package.json').version;
+const latestNPMJSSdkVersion = spawnOrFail('npm', [`show amazon-chime-sdk-js version`], { skipOutput: true }).trim();
```

Comment:
This `latestNPMJSSdkVersion` is been used in `release` script to update the JS SDK dependency. Now, this `npm show` command does not return `3.0.0-beta.1` which is published to NPM but `2.29.0` hence, the component library will be stuck at `2.x` versions, is this intended? Should we handle based on which version we are releasing either from `2.x` (current major) or `main` (next major -> 3.x) and accordingly get the beta tag for no?